src/coreComponents/physicsSolvers/multiphysics/HydrofractureSolver.cpp

```diff
@@ -209,10 +209,10 @@ real64 HydrofractureSolver< POROMECHANICS_SOLVER >::fullyCoupledSolverStep( real
     }
 
     // currently the only method is implicit time integration
-    dtReturn = nonlinearImplicitStep( time_n, dt, cycleNumber, domain );
+    dtReturn = nonlinearImplicitStep( time_n, dtReturn, cycleNumber, domain );
 
 
-    if( !this->m_performStressInitialization && m_surfaceGenerator->solverStep( time_n, dt, cycleNumber, domain ) > 0 )
+    if( !this->m_performStressInitialization && m_surfaceGenerator->solverStep( time_n, dtReturn, cycleNumber, domain ) > 0 )
```

Comment:
Simply change dt to dtReturn here may still have issues, as dt used in different fracture propagation iterations will still be different if timestep cut happens in some of the fracture propagation iteration.

Reply:
yeah, for the time being, i reverted it back to original formulation. Thanks.

Reply:
Yeah, I then saw the revert in the second commit